### PR TITLE
106 create name clarification field and create timestamp checkbox in signature modal

### PR DESCRIPTION
**Pull request by TyostoKarry**
During resolving merge conflicts, almost all the changes from commit "SignatureModal includes name clarification and timestamp" ended up on commit "Refactored opening modal"

#minor

### CODE REVIEW

File: client/src/components/SignatureModal.jsx [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 import MinutesContext from "../contexts/MinutesContext";
+import EditorContext from "../contexts/EditorContext";
 
-function SignatureModal({ open, onClose }) {
+function SignatureModal() {
   const theme = useTheme();
   const [, { updateMinutes }] = useContext(MinutesContext);
+  const [editor, updateEditor] = useContext(EditorContext);
+  const [signer, setSigner] = useState("");
+  const [isChecked, setIsChecked] = useState(true);
```
Comment (tuukkaviitanen) [inline client/src/components/SignatureModal.jsx:20]: isChecked should be descriptive. What is checked?

**Review by ArtoLaitinen — APPROVED** [review]

File: client/src/components/Editor.jsx [hunk 2]
```diff
@@ -147,12 +147,37 @@ function Editor() {
                     alt="Signature"
                   />
                 )}
-              <Typography variant="h5" sx={styles.signatureLine}>
+              <Box sx={styles.signatureAndDateLine} />
+              {state.minutes.signatures.length > 0 &&
+                !!state.minutes.signatures[0].signer && (
```
Comment (tuukkaviitanen) [inline client/src/components/Editor.jsx:152]: !! might be redundant. If necessary, use Boolean(value). You can read more here about that: https://www.ctomczyk.pl/stop-using-logical-not-and-casting-using-double-exclamation-mark-in-javascript-typescript/914/

**Review by tuukkaviitanen — CHANGES_REQUESTED** [review]
Few comments

**Review by tuukkaviitanen — APPROVED** [review]
Good job!

**Pull request merged**